ci/scripts/validate_testplans.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Use a JSON schema to validate the testplan hjson files have the correct structure.
+
+set -e
+
+dirs_with_testplan_files=(
+    "hw/top_earlgrey/data"
+    "hw/top_earlgrey/data/ip"
+)
+testplan_schema="hw/lint/sival_testplan_schema.hjson"
+
+for dir in "${dirs_with_testplan_files[@]}"; do
+    echo "Validating testplans under $dir:"
+    util/validate_testplans.py --dir "$dir" --schema "$testplan_schema" || {
+        echo -n "##vso[task.logissue type=error]"
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-error-message

Reply:
Ah, I see, thanks for the link. I wonder if I should retain this `##vso...` message for the Azure pipeline job to function correctly, and just add an additional error message for Github actions then? Something like:

```sh
echo -n "::error ::Failed testplan validation in ${dir}."
```

I wonder then if we need to change this in other scripts for the Github Actions CI to work as expected? See for example https://github.com/lowRISC/opentitan/blob/b120a77b03a706f32d64fdca6da429210b782252/ci/scripts/python-lint.sh#L48 and https://github.com/lowRISC/opentitan/blob/b120a77b03a706f32d64fdca6da429210b782252/ci/scripts/verible-lint.sh#L50 which I was looking at when I made this script originally.

Reply:
Great, thanks - that's very helpful.
I've added that now :+1: